fcrepo-kernel/src/main/java/org/fcrepo/kernel/utils/NamespaceTools.java

```diff
@@ -90,6 +94,12 @@ public static void validatePath(final Session session, final String path) {
         final String relPath = path.replaceAll("^/+", "").replaceAll("/+$", "");
         final String[] pathSegments = relPath.split("/");
         for (final String segment : pathSegments) {
+            // Empty path elements are not valid!
```

Comment:
It is difficult to agree or disagree with your statements without knowing exactly to which `those locations` or `this method` are referring. Would you please edit your comment to swap in the values for the references?